packages/build/tests/plugins/tests.js

```diff
@@ -142,13 +145,24 @@ test('Plugins can have inputs', async (t) => {
   t.snapshot(normalizeOutput(output))
 })
 
-test('Plugins are passed featureflags', async (t) => {
+test('Trusted plugins are passed featureflags and system log', async (t) => {
+  const systemLogFile = await tmpName()
   const output = await new Fixture('./fixtures/feature_flags')
     .withFlags({
       featureFlags: { test_flag: true },
+      debug: false,
+      systemLogFile: await fs.open(systemLogFile, 'a'),
     })
     .runWithBuild()
 
+  if (platform !== 'win32') {
```

Comment:
nit: It would be nice to add a comment explaining why we're excluding Windows.

Reply:
```suggestion
  // windows doesn't support the `/dev/fd/` API we're relying on for system logging.
  if (platform !== 'win32') {
```